### PR TITLE
feat: cleanup redundant normalizeAmount

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

`normalizeAmount` is an old implementation stemming from https://github.com/shapeshift/lib/pull/765/files#diff-6903517b56ea777f9301691a9a8b4801183fe6c04730ddb8e9d06d0c42fe5a8eR8, which was initially written for THOR swapper only and ended up being used in other swappers.

Following the implementation fix in https://github.com/shapeshift/web/pull/4894, this now removes it altogether.
We have the concept of base unit numbers, which are guaranteed to never contain decimal places - if they do, we have bigger problems. This makes this method redundant, thus the cleanup.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A, though relates to https://github.com/shapeshift/web/issues/4879 which will be closed when this goes in (we can also merge both as a stack to avoid double-testing)

## Risk

Theoretically none - if our base unit numbers are wrong, we have bigger problems.

## Testing

- Sanity check you can still get quotes for 1inch and ZRX and go to the next step (preview/approval)
- Ensure max. sell amounts are happy for said swappers

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

### ZRX

image
image
image

### 1inch

image
image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4894** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4894" target="_blank">Graphite</a> 
    * **PR #4895** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4895" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4895?utm_source=stack-comment).

File: src/lib/swapper/swappers/ZrxSwapper/getZrxTradeQuote/getZrxTradeQuote.test.ts [hunk 1]
```diff
@@ -62,8 +61,7 @@ jest.mock('context/PluginProvider/chainAdapterSingleton', () => {
 const mockOk = Ok as jest.MockedFunction<typeof Ok>
 const mockErr = Err as jest.MockedFunction<typeof Err>
 describe('getZrxTradeQuote', () => {
-  const sellAmount = '1000000000000000000'
-  ;(normalizeAmount as jest.Mock<string>).mockReturnValue(sellAmount)
+  // const sellAmount = '1000000000000000000'
```
Comment (kaladinlight) [inline src/lib/swapper/swappers/ZrxSwapper/getZrxTradeQuote/getZrxTradeQuote.test.ts:64]:
```suggestion
```

Comment (gomesalexandre) [reply src/lib/swapper/swappers/ZrxSwapper/getZrxTradeQuote/getZrxTradeQuote.test.ts]: eyy

**Review by kaladinlight — APPROVED** [review]

**Comment by gomesalexandre** [comment]
@gomesalexandre started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4895).

**Comment by gomesalexandre** [comment]
[Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4895) rebased this pull request as part of a merge.

**Comment by gomesalexandre** [comment]
@gomesalexandre merged this pull request with [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4895).

**Pull request merged**